operator/v1/tests/machineconfigurations.operator.openshift.io/IrreconcilableMachineConfig.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "MachineConfiguration"
+crdName: machineconfigurations.operator.openshift.io
+featureGates:
+  - IrreconcilableMachineConfig
+tests:
+  onCreate:
+    - name: Should be able to create a minimal MachineConfiguration
+      initial: |
+        apiVersion: operator.openshift.io/v1
+        kind: MachineConfiguration
+        spec: {} # No spec is required for a MachineConfiguration
+      expected: |
+        apiVersion: operator.openshift.io/v1
+        kind: MachineConfiguration
+        spec:
+          logLevel: Normal
+          operatorLogLevel: Normal
+    - name: Should be able to create an empty IrreconcilableMachineConfigValidationOverrides configuration knob
```

Comment:
Following https://github.com/openshift/api/pull/2433/files#r2263028988 this should be updated to _not_ allow this.